src/libostree/ostree-repo-refs.c

```diff
@@ -216,8 +217,8 @@ resolve_refspec_fallback (OstreeRepo     *self,
 
   if (self->parent_repo)
     {
-      if (!resolve_refspec (self->parent_repo, remote, ref,
-                            allow_noent, &ret_rev, error))
+      if (!resolve_refspec (self->parent_repo, remote, ref, allow_noent,
+                            TRUE, &ret_rev, error))
```

Comment:
Hm, yeah I think I let that slip, let me change that and make sure that it doesn't affect anything else, thanks for pointing it out!